data/cities/detroit/corporations/invictus.md

```diff
@@ -0,0 +1,20 @@
+name: Invictus
```

Comment:
Vendu pour : Bastion. L'ajout d'un complément type Technologies / Corporation / Enterprise / Industries / Incorporated est à discuter. C'est selon les envies.
Bastion seul est simple et efficace, les complément peuvent apporter une plus grande ouverture si on souhaite lui donner un aspect moins "boeuf".
Qui sonnerait aussi bien que Bastion seul pour moi serait Bastion Incorporated ou Bastion Industries
Initiales : B I : Be I : Soyez moi... message capilotracté certes mais ô combien lourd de sens.